src/main/resources/application.conf

```diff
@@ -27,7 +27,7 @@ spray.can.server {
 http {
   interface="0.0.0.0"
   port=8080
-  timeoutSeconds = 1
+  timeoutSeconds = 10
```

Comment:
I ran into a problem running locally on docker. It could have just been a blip but...